desktop/core/src/desktop/js/apps/newimporter/ImporterSourceSelector/ImporterSourceSelector.tsx

```diff
@@ -14,14 +14,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import React, { useRef } from 'react';
+import React, { useRef, useState } from 'react';
 import Button from 'cuix/dist/components/Button';
 import DocumentationIcon from '@cloudera/cuix-core/icons/react/DocumentationIcon';
+import S3Icon from '@cloudera/cuix-core/icons/react/S3Icon';
+import HDFSIcon from '@cloudera/cuix-core/icons/react/HDFSIcon';
```

Comment:
I think this breaks the CI build due to case sensitivity on the server but not on your local machine. The actual icon is found in @cloudera/cuix-core/icons/react/HdfsIcon.js but you are trying to import from @cloudera/cuix-core/icons/react/HDFSIcon.js with HDFS in all caps.